apps/software/lib/software/model/storage_drive.ex

```diff
@@ -33,17 +34,6 @@ defmodule Helix.Software.Model.StorageDrive do
   def create_changeset(params) do
```

Comment:
since an storage_drive is composed by the storage that contains it and the hdd it points to, this should receive both `storage_id` and `drive_id`